conda_smithy/configure_feedstock.py

```diff
@@ -1245,6 +1245,40 @@ def render_drone(jinja_env, forge_config, forge_dir, return_metadata=False):
     )
 
 
+def azure_build_id_from_token(forge_config):
```

Comment:
Please add docstrings to public functions

Reply:
Done.